benchmarks/Tracing/SpanCreationScenarios.cs

```diff
@@ -40,9 +40,9 @@ public static TelemetrySpan CreateSpan_Attributes(Tracer tracer)
         {
             var span = tracer.StartSpan("span");
             span.SetAttribute("attribute1", "1");
-            span.SetAttribute("attribute2", 2);
-            span.SetAttribute("attribute3", 3.0);
-            span.SetAttribute("attribute4", false);
+            span.SetAttribute("attribute2", "2");
```

Comment:
This may just be me still familiarizing myself with things, though why were the additional overloads of `SetAttribute` removed. Spec seems to suggest support for strings, booleans, numerics: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-attributes

Reply:
These will be added back when .NET Activity API added support for other data types in Tags. Should happen in Aug.

Reply:
Probably worth to put a TODO comment here @cijothomas?